strax/context.py

```diff
@@ -15,11 +14,17 @@
 export, __all__ = strax.exporter()
 
 
-# Placeholder value for omitted values.
-# Use instead of None since None might be a proper value/default
-OMITTED = '<OMITTED>'
-
-
+@strax.takes_config(
+    strax.Option(name='storage_converter', default=False,
```

Comment:
Will this mean that the strax key changes based on whether or not convertor?  Just wanting to make sure that it doesn't change lineage.

Reply:
It won't change the lineage. Context options are not tracked with the data, as they change how you load but not what you load.